src/hooks/useWindowWidth.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react';
+
+const useWindowWidth = () => {
+  const [lg, setLg] = useState(window.innerWidth >= 1450);
+
+  useEffect(() => {
+    const handleResize = () => {
+      setLg(window.innerWidth >= 1450);
+    };
+    window.addEventListener('resize', handleResize);
```

Comment:
You should also be nice and add `window.removeEventListener`
```suggestion
    window.addEventListener('resize', handleResize);
    () => window.removeEventListener('resize', handleResize);
```